vizro-core/docs/pages/user-guides/assets.md

```diff
@@ -87,6 +87,71 @@ For reference, see the [Vizro CSS files](https://github.com/mckinsey/vizro/tree/
     [AssetsCSS]: ../../assets/user_guides/assets/css_change.png
 
 
+## Overwrite CSS properties for specific pages
+Each page is assigned a unique identifier (`page.id`), which is sanitized and then stored as the ID of an outer `<div>` element.
+This identifier is formatted according to CSS naming conventions by removing non-alphanumeric characters and replacing spaces with dashes.
+For instance, if a `page.id` is "Variable Analysis," it will be stored as a CSS ID of "variable-analysis."
```

Comment:
```suggestion
For instance, if a `page.id` is `Variable Analysis` then it has CSS ID `variable-analysis`.
```

As it stands the "," and "." are very confusing here because it looks like you're replacing "," with ".".

Not sure whether we should convert to lowercase, but let's discuss 🤔

Reply:
Oh, I don't even know how this got in - probably my automatic grammar checker😆 There is not supposed to be a comma 😄